src/components/inputs/buttons/WorkflowButtons/WorkflowButtons.tsx

```diff
@@ -3,41 +3,53 @@ import './styles.css';
 function WorkflowButtons({
   showBack = true,
   showNext = true,
+  customNextStyle = {},
+  customBackStyle = {},
+  backText = 'Back',
+  nextText = 'Next',
   disableNext,
   disableBack,
   onNext,
   onBack,
 }: {
   showBack?: boolean;
   showNext?: boolean;
+  customNextStyle?: any;
+  customBackStyle?: any;
+  backText?: string;
+  nextText?: string;
```

Comment:
Yup, in workflow if no back or next text is defined it will pass undefined by default. If we make this required then we have to move the buttons logic to workflow, and i figured it was best to keep that logic in this component